packages/camera/camera_android/android/src/main/java/io/flutter/plugins/camera/Camera.java

```diff
@@ -209,6 +215,22 @@ public Camera(
         CameraFeatures.init(
             cameraFeatureFactory, cameraProperties, activity, dartMessenger, resolutionPreset);
 
+    if (null != fps && 0 < fps.intValue()) {
```

Comment:
Nit: here and below please don't reverse comparisons like this, as it's not our standard style and thus impairs legibility. Variables should be on the left, and constants on the right. (Modern compiler warnings generally make the issue these reversals were meant to solve moot.)